docs/self-managed/reference-architecture/kubernetes.md

```diff
@@ -0,0 +1,287 @@
+---
+id: kubernetes
+title: "Kubernetes deployment overview"
+sidebar_label: "Kubernetes"
+description: "Camunda 8 Kubernetes deployment Reference architecture home"
+---
+
+<!-- TODO: Update for 8.8 changes when it's clear what it means -->
+
+This reference architecture provides guidance on deploying Camunda 8 Self-Managed within a Kubernetes cluster. This deployment method is ideal for users who are looking to leverage the benefits of containerization and self-healing capabilities. It's well suited to quickly scale and easily provide resilience by utilizing multi-zone deployments.
+
+## Key features
+
+- **Scalability & High Availability**: Camunda 8 can scale up and down depending on your demand, and makes use of a highly available setup.
+- **Fault Tolerance & Resilience**: Scale Camunda 8 across zones or across regions to provide greater fault tolerance and uptime for your workflows.
+
+Although Kubernetes has a steeper learning curve, once properly configured, it provides significant advantages, including self-healing capabilities that ensure application resilience by automatically restarting failed containers and rescheduling workloads.
+
+Additionally, Kubernetes benefits from a vast ecosystem of **[Cloud Native Computing Foundation (CNCF) projects](https://www.cncf.io/)**, offering seamless integration with monitoring, observability, logging, and security solutions. Tools like Prometheus and Grafana allow for comprehensive monitoring, while service meshes such as Istio enhance traffic management and security.
+
+By leveraging this ecosystem, organizations can extend Kubernetes functionality to meet their specific operational needs, enabling greater automation, scalability, and visibility across their infrastructure.
+
+## Reference implementations
+
+This section includes deployment reference architectures for manual setups:
+
+- [Amazon EKS single-region](/self-managed/setup/deploy/amazon/amazon-eks/terraform-setup.md): a standard production setup.
+- [Amazon EKS dual-region](/self-managed/setup/deploy/amazon/amazon-eks/dual-region.md): an advanced use case utilizing two regions.
+
+For general deployment pitfalls, visit the [deployment troubleshooting guide](/self-managed/operational-guides/troubleshooting/troubleshooting.md).
+
+## Considerations
+
+- Most reference implementations primarily focus on the [Orchestration cluster](#components).
+- When scaling the web applications (Operate, Tasklist, Optimize), ensure that the importer/archiver runs only once within the whole cluster. See [high availability](#high-availability-ha) for for more information.
```

Comment:
I am not sure this is needed. We haven't yet really defined Orchestration cluster, and don't describe why this might be something worth considering. The importer/archiver note is clear in its section as well.

Reply:
you mean removing considerations as a whole? Fine by me.